test/controllers/EntryController.test.ts

```diff
@@ -0,0 +1,24 @@
+import 'reflect-metadata'
+import '../../src/controllers/EntryController'
+import { createApplication } from '../ApplicationFactory';
+import * as request from 'supertest'
+import { RedisService } from '../../src/services/RedisService';
+import { createSubstituteOf } from '../SubstituteFactory';
+import { MOVED_TEMPORARILY  } from 'http-status-codes';
+import { ENTRY_URI, PENALTY_DETAILS_PREFIX} from '../../src/utils/Paths';
+import { expect } from 'chai';
+
+describe('EntryController', () => {
+
+    describe('GET request', () => {
```

Comment:
The more I see this the more I like it. It gives me an idea what that controller is responsible for, what to expect.